client/components/QuestionsAndAnswers/MasterQA.jsx

```diff
@@ -1,12 +1,15 @@
-import React, { useState, useEffect } from 'react';
-import styled, { css } from 'styled-components';
+import React, { useState } from 'react';
+import styled from 'styled-components';
 import AddQuestion from './AddQuestion';
+import ModalForm from './ModalForm';
 import MoreQuestions from './MoreQuestions';
 import QuestionList from './QuestionList';
 import SearchQuestion from './SearchQuestion';
 import Data from './dummyData';
 
 // component
+const compareFn = (a, b) => b.question_helpfulness - a.question_helpfulness;
```

Comment:
Could `compareFn` be moved inside of MasterQA? No worries if not, just a thought!

Reply:
I'm debating about it, I might just move into the MasterQA since this func won't take too much space